src/django/pfb_analysis/serializers.py

```diff
@@ -27,7 +27,9 @@ def get_neighborhood_label(self, obj):
 
     class Meta:
         model = AnalysisJob
-        exclude = ('created_at', 'modified_at', 'created_by', 'modified_by', 'overall_scores',)
+        exclude = ('created_at', 'modified_at', 'created_by', 'modified_by', 'overall_scores',
+                   'analysis_job_definition', 'tilemaker_job_definition',
+                   '_analysis_job_name', '_tilemaker_job_name',)
```

Comment:
Went for it, then got exceptions. Added separate issue #282 to address later if time.